madmom/audio/ffmpeg.py

```diff
@@ -295,7 +295,10 @@ def get_file_info(infile, cmd='ffprobe'):
         if line.startswith(b'channels='):
             info['num_channels'] = int(line[len('channels='):])
         if line.startswith(b'sample_rate='):
-            info['sample_rate'] = int(line[len('sample_rate='):])
+            if cmd == 'avprobe':
+                info['sample_rate'] = int(float(line[len('sample_rate='):]))
```

Comment:
Does a single line `int(float(line[len('sample_rate='):]))` also work for all cases?
If yes, we could skip the if/else.

I'd like to have a test for this, but it is rather difficult. Because of the other avconv/avprobe issues, TravisCI uses the "real" ffmpeg, otherwise other tests fail.